# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../musepack-tools/gcc4.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- sv7/mppenc.c.orig	2005-03-05 17:38:00.000000000 +0200
+++ sv7/mppenc.c	2006-09-25 21:32:52.000000000 +0300
@@ -1864,7 +1864,7 @@
 }
 
 
-static void
+void
 OverdriveReport ( void )
 {
     if ( Overflows > 0 ) {                                                // report internal clippings
